apps/frontend/app/(client)/(main)/course/_components/CourseStatusTimeDiff.tsx

```diff
@@ -0,0 +1,122 @@
+'use client'
+
+import { cn } from '@/libs/utils'
+import ClockIcon from '@/public/icons/clock.svg'
```

Comment:
```suggestion
import clockIcon from '@/public/icons/clock.svg'
```
아이콘은 카멜케이스로 작성해주세요 (파스칼 케이스 아님)